src/test/java/com/znsio/teswiz/steps/JioCinemaSteps.java

```diff
@@ -40,4 +40,9 @@ public void iSwipeLeftOnTrendingInIndiaSection() {
     public void iAmAbleToViewNumberTrendingCinema(int movieNumberOnScreen) {
         new JioCinemaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).verifyMovieNumberVisibleOnScreen(movieNumberOnScreen);
     }
+
+    @When("I swipe {string} trending no {int} on trending in india section")
+    public void iSwipeRightTrendingNoOnTrendingInIndiaSection(String direction, int movieNumberOnScreen) {
+        new JioCinemaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).swipeMovieInTrendingInIndiaSection(direction,movieNumberOnScreen);
```

Comment:
swipeMovieInTrendingInIndiaSection. - method name should be swipeMovieTrendingInIndiaSection

Reply:
we are swiping the movie which is  in trending in india section